cartoframes/data/observatory/catalog.py

```diff
@@ -0,0 +1,13 @@
+from cartoframes.data.observatory.category import Categories
```

Comment:
_Adding the comment here, but it applies to the whole PR_

Let's try to use relative namespaces. It's a convention we are following and seems the more pythonic way of dealing with namespaces.

As a rule of thumb, internal module dependencies are relative and external are absolute. That means in our tests or examples we use absolute paths.

Reply:
Oh, got it. I thought absolute namespaces were the recommended ones (did I misread pep8 or is it a CARTO convention?) so I changed all of them to absolute at some point xD. 

Changed back!

Reply:
Yep we decided to use explicit relative imports for our internal modules based on this recommendation:

```
However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose
```